providers/ms365/resources/ms365.lr

```diff
@@ -95,15 +95,68 @@ microsoft.users {
   search string
 }
 
-// A container resource for identity and access policies that can be filtered.
+// A container resource for Identity and Access Governance features
 microsoft.identityAndAccess {
```

Comment:
this is a breaking change. Before `microsoft.identityAndAccess` returned a list of policies. Right now, you changed that to have 2 functions instead. This will break any existing queries related to this resource. We cannot ship such a change